main/src/main/java/org/mobilitydata/gtfsvalidator/notice/FeedExpirationDateNotice.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Google LLC, MobilityData IO
```

Comment:
If Google didn't contribute any code to this file, the copyright should just be MobilityData.